src/modules/shared/domain/valueObjects/email.vo.ts

```diff
@@ -24,14 +24,15 @@ export class Email extends ValueObject<EmailProps> {
     }
 
     public static async create(props: EmailProps): Promise<Email> {
+        const validatedProps = await Email.emailSchema.validate(props);
         try {
-            const validatedProps = await Email.emailSchema.validate(props);
             return new Email(validatedProps);
         } catch (error) {
             if (error instanceof ValidationError) {
+                const errorMessage = `Email Validation failed: ${validatedProps.value} is not a valid email address`;
```

Comment:
With that the handling of the EmailValidationError in the SamplesController of the mibi-portal-server probably does not work anymore. Needs to be tested and fixed.